pkg/platform/runtime/setup/buildlog/buildlog.go

```diff
@@ -216,6 +217,11 @@ func NewWithCustomConnections(artifactMap artifact.Map,
 					break
 				}
 
+				if _, ok := artifactsDone[m.ArtifactID]; ok {
+					multilog.Critical("buildlogstreamer sent duplicate ArtifactSucceededMessage for artifact %s", m.ArtifactID.String())
+					break
+				}
```

Comment:
Is this meant to be a part of another story?

Reply:
Huh, yeah.. I had run `git reset`, not sure why this is still in :\